tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -45,8 +44,12 @@ func (c *CustomClusterProvisioningTestSuite) SetupSuite() {
 
 	c.client = client
 
-	c.provisioningConfig.K3SKubernetesVersions, err = kubernetesversions.Default(c.client, clusters.K3SClusterType.String(), c.provisioningConfig.K3SKubernetesVersions)
-	require.NoError(c.T(), err)
+	if c.provisioningConfig.K3SKubernetesVersions == nil {
```

Comment:
So, is this not going to allow the default version to be selected if none is picked? Or does the list function just run all of the versions? I have this same question for all of the other files.

Reply:
so if you don't specify a k8s verison in your yaml, that's when we want to use the available versions from rancher. The if statement is there to allow for reading from the config.

Reply:
I see, can we test both of these scenarios just to make sure all is good?